.flake8

```diff
@@ -7,4 +7,6 @@ select = B,C,E,F,W,T4,B9,Q0,N8,VNE
 exclude =
     **migrations
     venv
-    tests
\ No newline at end of file
+    tests
+    settings.py
+    views.py
```

Comment:
Why you add views.py to exclude? NO! Remove it

Reply:
![Знімок екрана 2023-10-24 о 20 04 36](https://github.com/mate-academy/py-token-authentication/assets/134967594/2a022b24-d8b3-4a2d-8fad-cb179fe415ce)
because, I get error like this